cluster/src/test/java/org/apache/iotdb/cluster/integration/SingleNodeTest.java

```diff
@@ -46,6 +46,7 @@
   @Before
   public void setUp() throws Exception {
     super.setUp();
+    Thread.sleep(10000);
```

Comment:
Why sleep here?

Reply:
Thanks for your review, at that time, in order to test a CI, it has been removed now:D